pkg/account/component.go

```diff
@@ -51,8 +51,20 @@ func (c *component) UpdatePassword(ctx context.Context, currentPassword, newPass
 
 	_, err := c.keycloakClient.UpdatePassword(accessToken, realm, currentPassword, newPassword, confirmPassword)
 
+	var updateError error = nil
+	if err != nil {
+		switch err.Error() {
+		case "invalidPasswordExistingMessage":
+			updateError = commonhttp.Error{
```

Comment:
You can write it this way : updateError = commonhttp.CreateBadRequestError(err.Error())
But I'm not sure we want to send err.Error() in the response (could leak information as we are not sure what keycloak is returning)

Reply:
Yes, and related to this I did another PR for the Keycloak client, so that we don't send the whole response back but only the errorMessage part (if it is present)